app/models/guide.rb

```diff
@@ -13,4 +13,20 @@ class Guide < ActiveRecord::Base
   before_validation on: :create do |object|
     object.content_id = SecureRandom.uuid
   end
+
+  def work_in_progress_edition?
+    latest_edition.try(:state) == 'draft'
+  end
+
+  def update_attributes_from_params(params, state:)
```

Comment:
That makes a little more sense.

My thinking was that the underlying value of `state` can only be so many things. We don't want it to be an integer, `nil` or anything else. It should one of `draft` or `publishing`. Maybe we should make that clearer here if that's what you were trying to do?